lib/src/builders/permission_overwrite.dart

```diff
@@ -17,6 +17,7 @@ class PermissionOverwriteBuilder extends CreateBuilder<PermissionOverwrite> {
 
   @override
   Map<String, Object?> build() => {
+        'id': id.toString(),
```

Comment:
This is incorrect. While `PermissionOverwrite` objects do have an `id` field, this builder is only used in the [Edit Channel Permissions](https://discord.com/developers/docs/resources/channel#edit-channel-permissions), which does not take an `id` as a parameter (instead, it is passed in the route as you can see [here](https://github.com/nyxx-discord/nyxx/blob/main/lib/src/http/managers/channel_manager.dart#L491).